### ISSUE TITLE
Add ability to save screenshot

### ISSUE THREAD

**Issue by JohnVeness** [beatbrot/ScreenshotAssistant#14]
I understand the workflow of just using Share, but sometimes you just want to _save_ a screenshot. E.g. if you want to later upload it to a web-based forum that you cannot "share" to.

Please can you add a save feature, either as a separate button next to share, or within the Share list I suppose, although I would prefer a separate button.

Many thanks for creating this app.

**Comment by beatbrot** [comment]
Yeah I totally see the need for this. In [this](https://github.com/beatbrot/ScreenshotAssistant/issues/16#issuecomment-532369953) reply, I explain why its currently not included.

**Comment by kirisakow** [comment]
Hi and thanks for the app, from me too. It's very handy, really.

While I totally understand **your goal is to keep the UI very clean**, I am pretty sure the whole world would appreciate **an automatic save feature** with its config put away from main screen onto Settings screen and having a couple options (toggle the automatic save feature on and off (default), specify path where to save... just simple pleasures).

**Comment by beatbrot** [comment]
Sure thing! Unfortunately I don't have too much time for working on the app right now...

So I guess you have to wait quite a bit.... I am happy to accept PRs though :P

**Comment by KrasnayaPloshchad** [comment]
As a work around, you can try to share the screenshot to [Photo+Save](https://play.google.com/store/apps/details?id=cn.edu.buaa.wangye.photosave) app, which save an image directly into specific folder.

**Comment by kirisakow** [comment]
Thank you for the input. What I've been doing currently is share screenshots to the cloud (e.g. with NextCloud app) or to Telegram, which both are other ways to save.

**Comment by KrasnayaPloshchad** [comment]
And I saw another way to save, that is X-plore, which provides “Copy to” option in the share menu and let you choose the folder.
